### DISCUSSION TITLE
Permissions on new folder created with VSCode UI on a dev container

### DISCUSSION THREAD

**Discussion by rse-mfm** [devcontainers/community#156] [Q&A]
Hello,

When I create a new folder with VSCode UI inside a dev container, I do not have the right permissions on it (770).
What variables (or other) should I set in order to fix this ?

I've set umask already, but it doesn't seem to affect VSCode UI inside the dev container.

```
drwxrwx---  2 dev  dev_group 4096 Aug 26 09:33 test_1/   |--> created in SSH with VSCode UI
drwxr-xr-x  2 dev  dev_group 4096 Aug 26 09:34 test_2/   |--> created inside the dev container with VSCode UI
drwxrwx---  2 dev  dev_group 4096 Aug 26 09:34 test_3/   |--> created inside the dev container with the terminal
```

Thanks

**Comment by chrmarti** [comment]
Where do you set the umask?

**Reply by rse-mfm** [reply]
I've set the umask in` ~/.bashrc`, and then `/etc/profile`, `~/.profile`, no one worked. Any idea where I should set it ?
